bundles/org.eclipse.search/search/org/eclipse/search/internal/ui/text/FileLabelProvider.java

```diff
@@ -186,7 +187,7 @@ private int appendShortenedGap(String content, int start, int end, int charsToCu
 
 
 	private int getCharsToCut(int contentLength, Match[] matches) {
-		if (contentLength <= 256 || !"win32".equals(SWT.getPlatform()) || matches.length == 0) { //$NON-NLS-1$
+		if (contentLength <= 256 || !(Util.isWin32()) || matches.length == 0) { //$NON-NLS-1$
```

Comment:
Please change `!(Util.isWin32())` to be `!Util.isWin32()`

Reply:
done the changes.